spec/features/heroku_spec.rb

```diff
@@ -4,8 +4,9 @@
   scenario "Suspend a project for Heroku" do
     run_suspenders("--heroku=true")
 
-    expect(FakeHeroku).
-      to have_gem_included(project_path, "rails_stdout_logging")
+    %w(rails_stdout_logging).each do |gem|
```

Comment:
I was trying to keep the line < 80 chars and I thought that `have_gem_included` could be called more times in the future. Hound didn't like the `.\nto` too